CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+## Changelog ##
```

Comment:
I usually prefer [releases](https://github.com/benbalter/wordpress-github-sync/releases) over maintaining a text-based changelog. That way Git/tags can do the heavy lifting for us. Not strongly opinionated if you're up for maintaining it. :smile: